packages/binding-modbus/README.md

```diff
@@ -93,6 +93,12 @@ Along with content type `application/octet-stream`, this protocol binding accept
 
 **Note**: the list may be extended in the future. 
 
+In particular, the decimal number `9545 22880` will be encoded in as follows:
```

Comment:
- decimal format seems a bit strange...  4 digits followed by 5 digits ?
- I am not native speaker but isn't it "will be encoded in as follows" --> "will be encoded as follows"

Reply:
yes sorry, my bad they are actually *two* decimal numbers. Fixing.